docs/conf.py

```diff
@@ -296,3 +302,104 @@ def setup(app):
 if read_the_docs_build:
 
     subprocess.call('cd ../minorminer/docs/; make cpp', shell=True)
+
+# Link to GitHub source
+github_map = {'dwavebinarycsp': ['dwavebinarycsp',],
+              'cloud': ['dwave-cloud-client',],
+              'dimod':  ['dimod',],
+              'dwave_networkx': ['dwave-networkx',],
+              'greedy': ['dwave-greedy',],
+              'hybrid': ['dwave-hybrid',],
+              'inspector': ['dwave-inspector',],
+              'minorminer': ['minorminer',],
+              'neal': ['dwave-neal',],
+              'penaltymodel': [{'cache': ['penaltymodel_cache',],
+                               'core': ['penaltymodel_core',],
+                               'lp': ['penaltymodel_lp',],
+                               'maxgap': ['penaltymodel_maxgap',],
+                               'mip': ['penaltymodel_mip',]}],
+              'system': ['dwave-system',],
+              'embedding': ['dwave-system',],
+              'tabu': ['dwave-tabu',]}
```

Comment:
I'd simplify this to a `rtd-project-name` -> `gh-repo-name` map, and encode package versions separately (see below).

Reply:
Done